packages/examples/sdk-frontend-react/src/app/ChatUITest/MessageBubbles.tsx

```diff
@@ -19,24 +21,27 @@ export const MessageBubbles = () => {
         const pgpDecryptedPvtKey = await PUSHAPI.chat.decryptPGPKey({
             encryptedPGPPrivateKey: user.encryptedPrivateKey,
             signer: librarySigner,
-            env: ENV.STAGING
+            env: env
         })
 
         const ConversationHash = await PUSHAPI.chat.conversationHash({
-            account: account,
-            conversationId: '24b029b8e07e60291bf9d8c0c48ff993fa1e0a99105459f7404c425c92e91bac',
-            env: ENV.STAGING
+            account: `eip155:${account}`,
+            conversationId: 'eip155:0xEDF59F183584107B20e2c95189A4423224bba8F2',
+            env: env
         });
+        setConversationHash(ConversationHash.threadHash);
```

Comment:
@mishramonalisha76 can't we just pass the threadHash from it's response like ConversationHash.thredaHash in the chat.history api call ?